### PR TITLE
TOOLS-2470: Sign linux packages

### PR DESCRIPTION
**Pull request by varsha804**

### CODE REVIEW

**Comment by varsha804** [comment]
The signature of the release.rpm package has been verified:

Screen Shot 2020-03-05 at 4 20 41 PM

File: common.yml [hunk 1]
```diff
@@ -641,6 +641,37 @@ functions:
...
+      params:
+        working_dir: src/github.com/mongodb/mongo-tools/
+        silent: true
+        script: |
+          echo "${signing_auth_token_server_4_4}" > ./signing_auth_token
+    - command: shell.exec
+      type: system
+      params:
+        working_dir: src/github.com/mongodb/mongo-tools/
+        silent: true
```
Comment (rychipman) [inline common.yml]:
```suggestion
```
Let's not make this one silent, so that we can see if it prints out any messages!

File: common.yml [hunk 2]
```diff
@@ -641,6 +641,37 @@ functions:
...
+        script: |
+          /usr/local/bin/notary-client.py \
```
Comment (rychipman) [inline common.yml]:
```suggestion
          set -o errexit
          /usr/local/bin/notary-client.py \
```
Without this, the task won't actually fail if the first notary-client invocation errors, but the second one succeeds

File: common.yml [hunk 3]
```diff
@@ -641,6 +641,37 @@ functions:
...
+              --key-name "server-4.4" \
+              --auth-token-file ./signing_auth_token \
+              --comment "Evergreen Automatic Signing (mongo-tools)" \
+              --notary-url http://notary-service.build.10gen.cc:5000 \
+              --skip-missing \
+              --package-file-suffix "" \
```
Comment (rychipman) [inline common.yml]:
```suggestion
              --package-file-suffix "" \
              --outputs "sig" \
```
This was added to the msi signing in a recent commit on master, and we should do the same here. It makes sure that we only sign the package, and don't generate other extraneous artifacts

File: common.yml [hunk 4]
```diff
@@ -641,6 +641,37 @@ functions:
...
+              release.deb
+          /usr/local/bin/notary-client.py \
+              --key-name "server-4.4" \
+              --auth-token-file ./signing_auth_token \
+              --comment "Evergreen Automatic Signing (mongo-tools)" \
+              --notary-url http://notary-service.build.10gen.cc:5000 \
+              --skip-missing \
+              --package-file-suffix "" \
```
Comment (rychipman) [inline common.yml]:
```suggestion
              --package-file-suffix "" \
              --outputs "sig" \
```
same as above

File: common.yml (hunk 1 above)
Comment (varsha804) [reply common.yml]: Should we keep the `silent: true` in the `sign msi installer` task?

File: common.yml (hunk 2 above)
Comment (varsha804) [reply common.yml]: Should this also be set in the `sign msi installer` task?

Comment (pmeredit) [reply common.yml]: It should be set in an shell task unless we have a specific reason to *not* set it :)

**Pull request merged**